scss/_navbar.scss

```diff
@@ -32,6 +32,31 @@
     align-items: center;
     justify-content: space-between;
   }
+
+  .dropdown-menu {
```

Comment:
Why did you moved the `.dropdown-menu` selector under `.navbar` instead of `.navbar-nav`?